rollup.config.js

```diff
@@ -38,6 +38,14 @@ const options = [
     env: 'production',
     input: pkg.source,
   },
+  {
+    name,
+    umdName,
+    format: 'esm',
```

Comment:
Perhaps most users use webpack to bundle their app.
I think format may be ES module instead of CommonJS from the viewpoint of Tree Sharking.

IE11 support has been started in core lib with the following PR:

https://github.com/react-hook-form/react-hook-form/pull/235/files#diff-4dc771ce10b5894191688acc22e256b3

The core lib format is CommonJS, but I think this should also be fixed.